lib/pkgcloud/azure-v2/scripts/ssh.ps1

```diff
@@ -0,0 +1,8 @@
+$username = $args[0]
+$password = $args[1]
+$securePassword = ConvertTo-SecureString $password -AsPlainText -Force
+$credential = New-Object System.Management.Automation.PSCredential $username, $securePassword
+Enable-PSRemoting -Force
+$scriptPath = ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/DarwinJS/ChocoPackages/master/openssh/InstallChoco_and_win32-openssh_with_server.ps1'))
```

Comment:
Maybe it would be better to add this file to the repo or have a fork of the original repo so there are no surprises if it changes in the original repo